README.md

```diff
@@ -1,88 +1,126 @@
-# Twilio Video Quick Start for Swift
 
-> NOTE: This application uses Twilio's new Programmable Video Rooms API, which is currently available in Beta. The docs are [here](https://media.twiliocdn.com/sdk/ios/video/v1.0/docs/).
+[ ![Download](https://img.shields.io/badge/Download-1.0.0--beta14-blue.svg) ](https://www.twilio.com/docs/api/video/download-video-sdks#ios-sdk)
```

Comment:
Do we want to specify the version here? Every time we release we will need to come back to this repo and increment the version number, and the link still takes you to the same place.